src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportCreator;
+import core.basesyntax.service.TransactionMapper;
+import core.basesyntax.service.TransactionProcessor;
+import core.basesyntax.service.impl.CsvFileReader;
+import core.basesyntax.service.impl.CsvFileWriter;
+import core.basesyntax.service.impl.ReportCreatorImpl;
+import core.basesyntax.service.impl.TransactionMapperImpl;
+import core.basesyntax.service.impl.TransactionProcessorImpl;
+import core.basesyntax.service.strategy.BalanceOperation;
+import core.basesyntax.service.strategy.OperationStrategy;
+import core.basesyntax.service.strategy.PurchaseOperation;
+import core.basesyntax.service.strategy.ReturnOperation;
+import core.basesyntax.service.strategy.SupplyOperation;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String SOURCE = "src/main/resources/source.csv";
```

Comment:
Better specify that this is path to file, `source` can mean many things.
```suggestion
    private static final String SOURCE_PATH = "src/main/resources/source.csv";
```